srcs/main.c

```diff
@@ -6,19 +6,19 @@
 /*   By: sunpark <sunpark@student.42seoul.kr>       +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2020/11/22 22:05:59 by sunpark           #+#    #+#             */
-/*   Updated: 2021/01/02 16:03:13 by hyukim           ###   ########.fr       */
+/*   Updated: 2021/01/04 20:47:36 by hyukim           ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "minishell.h"
 
-int				get_source()
+int				get_source(void)
 {
 	char		**cmd;
 	int			cmd_status;
 
-	cmd = &(g_bash->src.buf);
-	cmd_status = get_command(cmd);
+	cmd = &(g_bash->cmd);
+	cmd_status = get_command();
```

Comment:
```suggestion
	cmd_status = get_command();
```

이 함수에서 사용하는 `cmd` 역시 `g_bash->cmd`로 교체 가능한 것 같습니다.

Reply:
넵 그러네요. 요청하신 부분 처리했습니다!